src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -8,14 +8,14 @@
       </p>
     </header>
 
-    <!-- Dissolution -->
+    <!-- Dissolution summary -->
     <v-card flat id="dissolution-summary" class="mt-8">
       <header class="review-header rounded-t">
         <v-icon class="ml-2" color="appDkBlue">mdi-domain-remove</v-icon>
         <label class="font-weight-bold pl-2">Dissolution</label>
       </header>
 
-      <div  class="pb-8" :class="{ 'invalid-section': isDefineDissolutionInvalid }">
+      <div class="pb-8" :class="{ 'invalid-section rounded-bl-0': isDefineDissolutionInvalid }">
```

Comment:
This is a div inside a v-card, so this special class is needed to square off the bottom left error border.